sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/models/QueueDescription.java

```diff
@@ -141,6 +141,11 @@ private static final class AuthorizationRulesWrapper {
             namespace = "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect")
         private final List<AuthorizationRuleImpl> items;
 
+        @JsonCreator
+        private AuthorizationRulesWrapper() {
+            this.items = new ArrayList<AuthorizationRuleImpl>();
```

Comment:
Could this be replaced with `Collections.emptyList()` so we don't initialise a new array?